plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -205,10 +206,10 @@ test.describe.serial( 'Add New Variable Product Page', () => {
 			if ( i > 0 ) {
 				await page.click( 'button.add_attribute' );
 			}
-			await page.fill(
-				`input[name="attribute_names[${ i }]"]`,
-				`attr #${ i + 1 }`
-			);
+			const input = `input[name="attribute_names[${ i }]"]`;
+
+			await page.waitForSelector( input, { timeout: 1000 } ); // Wait for up to 1 seconds
+			await page.fill( input, `attr #${ i + 1 }` );
```

Comment:
I think you can do something like...

```
await page.locator( `input[name="attribute_names[${ i }]"]` ).fill( `attr #${ i + 1 }` );
```

It should auto-wait for you.

Why do you need the explicit  timeout on the wait?